leaper.py

```diff
@@ -221,7 +221,6 @@ def check_source_submission(self, src_project, src_package, src_rev, target_proj
                     self.needs_release_manager = True
                 if origin == src_project:
                     self.source_in_factory = True
-                    return True
```

Comment:
What effect does this have? Does not seem to fall to logic with additional links.